server/executor/trigger/triggerer.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type Triggerer interface {
-	Trigger(context.Context, model.Test, trace.TraceID, trace.SpanID) (Response, error)
+	Trigger(context.Context, context.Context, model.Test, trace.TraceID, trace.SpanID) (Response, error)
```

Comment:
It's really confusing to have 2 contexts here, so it would be great to name these params so we can know what is which